packages/sil-governance/src/routes/create-proposal/components/FormOptions/DistributeFunds/index.tsx

```diff
@@ -59,6 +60,10 @@ const DistributeFunds = ({ form, recipientsChanged }: Props): JSX.Element => {
         setRecipients(recipients);
         recipientsChanged(recipients);
       } catch (error) {
+        // TODO the value of an input file can only be set to empty string, not working
+        (document.getElementById(IMPORT_FIELD_ID) as HTMLInputElement).value = "";
```

Comment:
The issue with this approach is that it only changes the rendered html component, not the state that remains unchanged in the React component. Let's better not access DOM directly like this